src/bundle/Core/Command/VirtualFieldDuplicateFixCommand.php

```diff
@@ -0,0 +1,272 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\Core\Command;
+
+use Doctrine\DBAL\Connection;
+use Exception;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+final class VirtualFieldDuplicateFixCommand extends Command
+{
+    private const DEFAULT_BATCH_SIZE = 10000;
+
+    private const MAX_ITERATIONS_UNLIMITED = -1;
+
+    private const DEFAULT_SLEEP = 0;
+
+    /** @var \Doctrine\DBAL\Connection */
+    private $connection;
+
+    public function __construct(
+        Connection $connection
+    ) {
+        parent::__construct('ibexa:content:remove-duplicate-fields');
+        $this->setDescription('Removes duplicate fields created as a result of faulty IBX-5388 performance fix.');
+
+        $this->connection = $connection;
+    }
+
+    public function configure(): void
+    {
+        $this->addOption(
+            'batch-size',
+            'b',
+            InputOption::VALUE_OPTIONAL,
+            'Number of attributes affected per iteration',
+            self::DEFAULT_BATCH_SIZE
+        );
+
+        $this->addOption(
+            'max-iterations',
+            'i',
+            InputOption::VALUE_OPTIONAL,
```

Comment:
This option, once passed, needs to have value.
```suggestion
            InputOption::VALUE_REQUIRED,
```